tests/034_plugin.ts

```diff
@@ -1,7 +1,7 @@
 const { openPlugin, pluginFilename, env } = Deno;
 
 const plugin = openPlugin(
-  env().DENO_BUILD_PATH + "/" + pluginFilename("test_plugin")
+  env().DENO_BUILD_PATH + "/rust_crates/" + pluginFilename("test_plugin")
```

Comment:
This is unfortunate...

Reply:
It is, but it works. Any idea what is up with the appveyor build?

Reply:
`tools/build.py --release` works fine on windows, but `tools/build.py` doesn't for some reason.